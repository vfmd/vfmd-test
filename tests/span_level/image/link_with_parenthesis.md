
![bad link](url) "title")

![bad link](url\) "title")

![link](<url)> "title")
