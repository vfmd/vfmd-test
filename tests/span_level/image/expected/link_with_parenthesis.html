<p><img src="url" alt="bad link" /> &quot;title&quot;)</p>

<p><img src="url%5C" alt="bad link" /> &quot;title&quot;)</p>

<p><img src="url)" alt="link" title="title" /></p>

